webapp-django/crashstats/profile/templates/profile/profile.html

```diff
@@ -0,0 +1,198 @@
+{% extends "crashstats_base.html" %}
+
+{% block page_title %}Your Profile{% endblock %}
+
+{% block site_css %}
+    {{ super() }}
+
+    {% compress css %}
+<link type="text/css" rel="stylesheet" href="{{ static('profile/css/profile.css') }}">
+    {% endcompress %}
+
+{% endblock %}
+
+{% block site_js %}
+    {{ super() }}
+
+<script src="{{ static('crashstats/js/moment.min.js') }}"></script>
+
+    {% compress js %}
+<script type="text/javascript" src="{{ static('crashstats/js/socorro/your_crashes.js') }}"></script>
+    {% endcompress %}
+
+    {% compress js %}
+<script type="text/javascript" src="{{ static('crashstats/js/timeutils.js') }}"></script>
+<script type="text/javascript" src="{{ static('tokens/js/home.js') }}"></script>
+    {% endcompress %}
+
+{% endblock %}
+
+{% block content %}
+
+<div id="mainbody">
+
+    <div class="page-heading">
+        <p class="profile-email">
+            {{ request.user.email }}
+        </p>
+        <h2>Your Profile</h2>
+    </div>
+
+    <div class="panel">
+        <div class="title">
+            <h2>Your permissions</h2>
+        </div>
+        <div class="body">
+        {% if request.user.is_superuser %}
+            <p>
+                You are a <b>superuser</b>. You have <b>unrestricted access to everything</b> on the <a href="{{ url('manage:home') }}">admin</a> page.
+            </p>
+        {% else %}
+        <table class="data-table">
+            <thead>
+                <tr>
+                    <th>Permission</th>
+                    <th>You</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for permission in permissions %}
+                <tr>
+                    <td>{{ permission.name }}</td>
+                    <td>
+                    {% if request.user.has_perm('crashstats.' + permission.codename) %}
+                        Yes!
+                    {% else %}
+                        No
+                    {% endif %}
+                </tr>
+                  {% endfor %}
+            </tbody>
+        </table>
+        {% endif %}
+        </div>
+    </div>
+
+    <div class="page-section">
+        <div class="panel">
+            <div class="title">
+                <h2>Your crash reports</h2>
```

Comment:
The title of the page is "Your Profile" (with a capital P). Then this should be "Your Crash Reports". Or, the title should maybe be "Your profile". Either way, it should be consistent.